actions/src/main/java/com/redhat/service/bridge/actions/webhook/WebhookInvoker.java

```diff
@@ -0,0 +1,46 @@
+package com.redhat.service.bridge.actions.webhook;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.bridge.actions.ActionInvoker;
+
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class WebhookInvoker implements ActionInvoker {
+
+    public static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
+    private static final Logger LOG = LoggerFactory.getLogger(WebhookInvoker.class);
+
+    private final String endpoint;
+    private final OkHttpClient client;
```

Comment:
@kostola Any reason not to use the VertX WebClient here rather than the OKHttp dependency? See: https://quarkus.io/guides/vertx#using-vert-x-clients

Reply:
I used OkHttp because the dependency is already in the Quarkus BOM, but fine for me to switch to VertX if that's what you prefer. Wdyt @danielezonca ?

Reply:
Switched to VertX WebClient :+1: